packages/core/test/integration/support.ts

```diff
@@ -1,19 +1,17 @@
-import assert from 'node:assert';
 import pTimeout from 'p-timeout';
 import { QueryTypes } from '@sequelize/core';
 import type { AbstractQuery } from '@sequelize/core/_non-semver-use-at-your-own-risk_/dialects/abstract/query.js';
 import * as Support from '../support';
```

Comment:
Not really in the scope of this PR, but can we only import the methods we're actually using here?

Reply:
Done